example/lib/main.dart

```diff
@@ -41,7 +41,11 @@ class _MyHomePageState extends State<MyHomePage> {
     super.initState();
 
     // Initialize OAuth library
-    oauth.init().then((value) => _oauthInitialized = true);
+    oauth.init().then((value) {
```

Comment:
why not in main function with async and await? then when you reach this screen, it's already initialized